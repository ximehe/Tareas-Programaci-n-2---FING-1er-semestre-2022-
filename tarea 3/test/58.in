 # implementar caminoAscendente
insertarEnAbb (20,20.00)
insertarEnAbb (15,25.0)
insertarEnAbb (10,10.0)
insertarEnAbb (40,11.0)
insertarEnAbb (50,5.0)
insertarEnAbb (30,15.0)
insertarEnAbb (25,6.0)
insertarEnAbb (35,10.0)
insertarEnAbb (32,5.0)
insertarEnAbb (27,4.0)
imprimirAbb
caminoAscendente 20 0
caminoAscendente 20 1
caminoAscendente 20 2
caminoAscendente 40 0
caminoAscendente 40 1
caminoAscendente 40 2
caminoAscendente 40 3
caminoAscendente 15 0
caminoAscendente 15 1
caminoAscendente 15 2
caminoAscendente 15 3
caminoAscendente 50 0
caminoAscendente 50 2
caminoAscendente 50 4
caminoAscendente 10 3
caminoAscendente 25 3
caminoAscendente 35 0
caminoAscendente 35 2
caminoAscendente 32 0
caminoAscendente 32 4
caminoAscendente 32 5
caminoAscendente 32 6
Fin