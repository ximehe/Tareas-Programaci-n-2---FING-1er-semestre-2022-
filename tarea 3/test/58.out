1>#  implementar caminoAscendente.
2>Se insertó (20,20.00) en el abb.
3>Se insertó (15,25.00) en el abb.
4>Se insertó (10,10.00) en el abb.
5>Se insertó (40,11.00) en el abb.
6>Se insertó (50,5.00) en el abb.
7>Se insertó (30,15.00) en el abb.
8>Se insertó (25,6.00) en el abb.
9>Se insertó (35,10.00) en el abb.
10>Se insertó (32,5.00) en el abb.
11>Se insertó (27,4.00) en el abb.
12>
--(50,5.00)
-(40,11.00)
---(35,10.00)
----(32,5.00)
--(30,15.00)
----(27,4.00)
---(25,6.00)
(20,20.00)
-(15,25.00)
--(10,10.00)
13>
La posición actual NO está definida.
14>20 
La posición actual NO está definida.
15>20 
La posición actual NO está definida.
16>
La posición actual NO está definida.
17>40 
La posición actual NO está definida.
18>40 20 
La posición actual NO está definida.
19>40 20 
La posición actual NO está definida.
20>
La posición actual NO está definida.
21>15 
La posición actual NO está definida.
22>15 20 
La posición actual NO está definida.
23>15 20 
La posición actual NO está definida.
24>
La posición actual NO está definida.
25>50 40 
La posición actual NO está definida.
26>50 40 20 
La posición actual NO está definida.
27>10 15 20 
La posición actual NO está definida.
28>25 30 40 
La posición actual NO está definida.
29>
La posición actual NO está definida.
30>35 30 
La posición actual NO está definida.
31>
La posición actual NO está definida.
32>32 35 30 40 
La posición actual NO está definida.
33>32 35 30 40 20 
La posición actual NO está definida.
34>32 35 30 40 20 
La posición actual NO está definida.
35>Fin.
